mitodl/hugo-course-publisher
# Fix relative links to static content

#### Pre-Flight checklist

#### What are the relevant tickets?
Partially resolves https://github.com/mitodl/ocw-to-hugo/issues/22

#### What's this PR do?
Adds updated markdown with fixed links to static content

#### How should this be manually tested?
Browse to the various courses and make sure all links to static content (PDF's, images, assignment file downloads) work as expected
